websocket_server/SocketManager.js

```diff
@@ -23,7 +23,7 @@ export default class SocketManager {
 		this.socketDataManager = new SocketDataManager(storageManager)
 
 		this.io = new SocketIO(server, {
-			transports: ['websocket'],
+			transports: ['websocket', 'polling'],
```

Comment:
Is this related?

Reply:
Yes, it is actually a fallback as vitest was failing on the websocket handling (didn't dig too deep into that, but polling is the regular fallback mechanism and would also be needed for #119)